src/util/triplea/MapXMLCreator/CanalDefinitionsPanel.java

```diff
@@ -0,0 +1,499 @@
+package util.triplea.MapXMLCreator;
+
+import games.strategy.util.Tuple;
+
+import java.awt.Color;
+import java.awt.FontMetrics;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.Point;
+import java.awt.event.ActionEvent;
+import java.awt.event.MouseEvent;
+import java.awt.geom.Rectangle2D;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.LinkedHashSet;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.SortedSet;
+import java.util.TreeSet;
+
+import javax.swing.AbstractAction;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+import javax.swing.SwingUtilities;
+
+import util.triplea.MapXMLCreator.TerritoryDefinitionDialog.DEFINITION;
+
+/**
+ * 
+ * @author Erik von der Osten
+ * 
+ */
+public class CanalDefinitionsPanel extends ImageScrollPanePanel
+{
+	protected static final String NEW_CANAL_OPTION = "<new Canal>";

```

Comment:
I have uploaded another commit here accidentally (https://github.com/triplea-game/triplea/pull/485/commits)
However I think I managed to push that commit to my branch MapXMLCreator_Java (https://github.com/frigoref/triplea/tree/MapXMLCreator_Java). Could you please help me verify that?